VERSION

```diff
@@ -1 +1 @@
-0.30.1
+0.31.1
```

Comment:
we've just released 0.30.0; if it'll land (and get into bumpenvs) by EOD today, you can keep it at 0.30.1. otherwise it'll probably be 0.30.2